### PR TITLE
Remove unused vars

### PR DESCRIPTION
**Pull request by limonte**
Connected to #2711

While searching for the tool to remove the unused methods, I figured that we have some unused vars and that it's easy to clean them up with eslint.

### CODE REVIEW

File: extension/js/content_scripts/webmail/gmail-element-replacer.ts [hunk 1]
```diff
@@ -259,14 +258,13 @@ export class GmailElementReplacer implements WebmailElementReplacer {
       for (const attsContainerEl of $(this.sel.attsContainerInner)) {
         const attsContainer = $(attsContainerEl);
         const newPgpAtts = this.filterAtts(attsContainer.children().not('.evaluated'), Att.webmailNamePattern).addClass('evaluated');
-        const newPgpAttsNames = Browser.arrFromDomNodeList(newPgpAtts.find('.aV3')).map(x => $.trim($(x).text()));
```
Comment (limonte) [inline extension/js/content_scripts/webmail/gmail-element-replacer.ts:262]: this wasn't used anywhere

File: extension/js/common/platform/catch.ts [hunk 2]
```diff
@@ -290,7 +290,7 @@ export class Catch {
   }
 
   private static formattedStackBlock = (name: string, text: string) => {
-    return `\n\n### ${name} ###\n# ${Catch.stackTrace().split('\n').join('\n# ')}\n######################\n`;
+    return `\n\n### ${name} ###\n# ${text.split('\n').join('\n# ')}\n######################\n`;
```
Comment (limonte) [inline extension/js/common/platform/catch.ts:293]: caught a tiny bug here

File: test/source/platform/store/contact-store.ts [hunk 3]
```diff
@@ -13,7 +13,7 @@ export class ContactStore {
     return result;
   }
 
-  public static obj = async ({ email, name, client, pubkey, pendingLookup, lastUse, lastCheck, lastSig }: any): Promise<Contact> => {
+  public static obj = async ({ email, name, client, pubkey }: any): Promise<Contact> => {
```
Comment (tomholub) [inline test/source/platform/store/contact-store.ts:16]: I'll have to have a closer look at this

**Review by tomholub — COMMENTED** [review]
Looks good, thanks!

File: extension/js/common/platform/catch.ts (hunk 2 above)
Comment (tomholub) [reply extension/js/common/platform/catch.ts:293]: nice :+1:

**Review by tomholub — APPROVED** [review]

**Pull request merged**